src/main/scala/com/gd/twitteranalytics/reports/ExternalConfigLoader.scala

```diff
@@ -5,6 +5,7 @@ import java.util.Properties
 
 import com.gd.twitteranalytics.util.AppConfigReader
 
+//TODO Check the external configuration feasability with deployment to Oozie workflow manager..
```

Comment:
NOTE: I noticed that you like to put double dots in the end of sentences - which is ok for personal/informal communication, but it doesn't seem to be ok for code documentation and comments, since they should be more formal.

Reply:
Fixed.